scenes/lessons/7th_class/tarayin_artahaytutyun/tarayin_artayahtutyun.py

```diff
@@ -0,0 +1,305 @@
+        road_length = road.get_right()[0] - road.get_left()[0]
+        road_endpoint_left = Dot(road.get_left())
+        road_endpoint_right = Dot(road.get_right())
+
+        brace_total = Brace(road, DOWN)
+        total_distance = Tex('700', tex_template=ARMTEX).next_to(brace_total, DOWN)
+
+        car = SVGMobject('../../../../objects/SVG_files/small_car').set_color(RED).scale(0.2).move_to(road_endpoint_left)
+        car_trace = always_redraw(lambda: Line(road_endpoint_left.get_center(), car.get_center(), color=GREEN, stroke_width=8))
+
+        speed_vector = VGroup(Line().scale(0.25).rotate(PI/2).next_to(car, UP))
+        speed_vector.add(Line().scale(0.25).next_to(speed_vector[0], UP, buff=-0.02, aligned_edge=LEFT))
+        speed_vector.add(Arrow().tip.scale(0.5).next_to(speed_vector[1], RIGHT, buff=0))
+        speed_vector.add(Tex('60', font_size=35).next_to(speed_vector[1], UP))
+        
+        brace_passed = BraceBetweenPoints(road_endpoint_left.get_center(), road_endpoint_left.get_center() + 1.72 * RIGHT, UP)
+        passed_distance = Tex('240', tex_template=ARMTEX).next_to(brace_passed, UP)
+
+        brace_remaining = BraceBetweenPoints(road_endpoint_left.get_center() + 1.72 * RIGHT, road_endpoint_right.get_center(), UP)
+        harcakan = Tex('?').next_to(brace_remaining, UP)
+        remaining_distance = Tex('460', tex_template=ARMTEX).next_to(brace_remaining, UP)
+
+    # լուծում
+        solution_first_step = Tex('1) ', '60', ' $\cdot$ ', '4', ' = ', '240', tex_template=ARMTEX)
+        solution_first_step.to_edge(LEFT).shift(DOWN)
+
+        solution_second_step = Tex('2) ', '700', ' - ', '240', ' = ', '460', tex_template=ARMTEX)
+        solution_second_step.to_edge(LEFT).shift(2 * DOWN)
+    
+    # լուծումն արտահայտությամբ ժամանակի տարբեր արժեքների դեպքում
+        solution_4 = Tex('700', ' - ', '60', ' $\cdot$ ', '4', ' = ', '460')
+        solution_4.next_to(VGroup(solution_first_step, solution_second_step), RIGHT).shift(2.5 * RIGHT)
+
+        solution_5_final = Tex('700', ' - ', '60', ' $\cdot$ ', '5', ' = ', '400')
+        solution_5_final.move_to([0, -0.75, 0])
+
+        solution_5_initial = Tex('700', ' - ', '60', ' $\cdot$ ', '4')
+        solution_5_initial.move_to(solution_5_final)
+        solution_5_initial.align_to(solution_5_final, LEFT)
+
+        solution_7_final = Tex('700', ' - ', '60', ' $\cdot$ ', '7', ' = ', '280')
+        solution_7_final.move_to([0, -1.5, 0])
+
+        solution_7_final_copy = solution_7_final.copy()
+        case_t_is_7 = VGroup(
+            Tex('t = 7'),
+            Tex('դեպքում', tex_template=ARMTEX)
+        )
+        case_t_is_7.arrange(buff=0.2, aligned_edge=UP).next_to(solution_7_final_copy, LEFT, buff= 0.5)
+
+        solution_7_initial = Tex('700', ' - ', '60', ' $\cdot$ ', '5')
+        solution_7_initial.move_to(solution_7_final)
+        solution_7_initial.align_to(solution_7_final, LEFT)
```

Comment:
I think you can write a mini function for this, you can even do inside this function, so you wouldn't have to scroll.

Reply:
միգուցե ։դ